test/e2e/node_feature_discovery.go

```diff
@@ -545,7 +153,7 @@ var _ = SIGDescribe("Node Feature Discovery", func() {
 				// Launch nfd-worker
 				By("Creating a nfd worker pod")
 				image := fmt.Sprintf("%s:%s", *dockerRepo, *dockerTag)
-				workerPod := nfdWorkerPod(image, []string{"-oneshot", "-label-sources=fake"})
+				workerPod := testutils.NFDWorkerPod(image, []string{"--oneshot", "--label-sources=fake"})
```

Comment:
nit: don't change arguments (back) to double dash

```suggestion
				workerPod := testutils.NFDWorkerPod(image, []string{"-oneshot", "-label-sources=fake"})
```

Reply:
sure thing, fixing